src/main/java/Paimon.java

```diff
@@ -0,0 +1,180 @@
+package main.java;
+
+import main.java.task.*;
+
+import java.util.Scanner;
+
+public class Paimon {
+    /**
+     * Default function for sending messages, includes text formatting
+     */
+    private static void sendTaskMessage(String mainMessage, String subMessage, String closingMessage) {
+        System.out.println(mainMessage);
+        System.out.println("-------------------->");
+        System.out.println(subMessage);
+        System.out.println("-------------------->");
+        System.out.println(closingMessage);
+    }
+
+    private static void sendTaskMessage(String mainMessage, String subMessage) {
+        System.out.println(mainMessage);
+        System.out.println("-------------------->");
+        System.out.println(subMessage);
+        System.out.println("-------------------->");
+    }
+
+    private static void printTaskList(TaskList list) {
+        if (list.getSize() == 0) {
+            System.out.println("Your list is currently empty, add some tasks!");
+        } else {
+            System.out.println("Here is your list so far!");
+            System.out.println("-------------------->");
+            System.out.println(list);
+            System.out.println("-------------------->");
+        }
+
+    }
+
+    private static void greeting() {
+        System.out.println("-------------------->" + "\nGreetings Traveller!\nI'm Paimon, your friendly guide!\nType help to see what I can help you with!" + "\n-------------------->");
+    }
+
+    private static void goodbye() {
+        System.out.println("-------------------->" + "\nSee you later Traveller!" + "\n-------------------->");
+    }
+
+
+    public static void main(String[] args) {
+        greeting();
+        Scanner scanner = new Scanner(System.in);
+        TaskList taskList = FileHandler.loadTaskList();
+        boolean isActive = true;
+        while (isActive) {
```

Comment:
I think the main loop is pretty long (over 100 lines). Would it be possible to shift the logic elsewhere?